src/KK.AspNetCore.EasyAuthAuthentication/EasyAuthAuthenticationHandler.cs

```diff
@@ -14,53 +14,63 @@ namespace KK.AspNetCore.EasyAuthAuthentication
     using Newtonsoft.Json;
     using Newtonsoft.Json.Linq;
 
+    /// <summary>
+    /// Enables the handler in an Easy Auth context.
+    /// </summary>
     public class EasyAuthAuthenticationHandler : AuthenticationHandler<EasyAuthAuthenticationOptions>
     {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="EasyAuthAuthenticationHandler"/> class.
+        /// </summary>
+        /// <param name="options">An instance of <see cref="EasyAuthAuthenticationOptions"/>.</param>
+        /// <param name="logger">An instance of <see cref="ILoggerFactory"/>.</param>
+        /// <param name="encoder">An instance of <see cref="UrlEncoder"/>.</param>
+        /// <param name="clock">An instance of <see cref="ISystemClock"/>.</param>
         public EasyAuthAuthenticationHandler(
             IOptionsMonitor<EasyAuthAuthenticationOptions> options,
             ILoggerFactory logger,
             UrlEncoder encoder,
-            ISystemClock clock
-        ) : base(options, logger, encoder, clock)
+            ISystemClock clock) : base(options, logger, encoder, clock)
         {
-
         }
 
+        /// <inheritdoc/>
```

Comment:
here is the doc of the function AuthenticationHandler<T>.HandleAuthenticateAsync()???

Reply:
I saw this on a Microsoft repo but I don not know if this works.